README.md

```diff
@@ -44,10 +44,10 @@ this will install the required julia packages. This step is only necessary the f
 # How to run an example
 Assuming you are on the folder RamirezReyes_ShallowWaterInFPlane, use bash to launch julia in the following way:
 
-`julia --project=@. -t 32 -e scripts/examples/run_example_cpu.jl`
+`julia --project=@. -t 32 -e scripts/examples/run_oceananigans_example_cpu.jl`
```

Comment:
Would it make sense to drop the thread count in the example to make sure more people are able to run it? Would 4-8 be acceptable?